lib/xnvme_be_cbi_async_emu.c

```diff
@@ -26,14 +26,14 @@ struct _emu_entry {
 	uint32_t meta_vec_cnt;
 	uint32_t is_vectored;
 
-	STAILQ_ENTRY(_emu_entry) link;
+	STAILQ_ENTRY(qpair_entry) link;
 };
 
-struct _emu_qp {
-	STAILQ_HEAD(, _emu_entry) rp; ///< Request pool
-	STAILQ_HEAD(, _emu_entry) sq; ///< Submission queue
+struct qpair {
+	STAILQ_HEAD(, qpair_entry) rp; ///< Request pool
+	STAILQ_HEAD(, qpair_entry) sq; ///< Submission queue
 	uint32_t capacity;
-	struct _emu_entry elm[];
+	struct qpair_entry elm[];
 };
 
```

Comment:
During compilation I see a lot of warnings

lib/xnvme_be_cbi_async_emu.c:49:37: note: expected ‘struct qpair *’ but argument is of type ‘struct _emu_qp *’
   49 | int static qpair_term(struct qpair *qp)

lib/xnvme_be_cbi_async_emu.c:94:25: warning: passing argument 1 of ‘qpair_alloc’ from incompatible pointer type [-Wincompatible-pointer-types]
   94 |         if (qpair_alloc(&(queue->qp), queue->base.capacity)) {

and so on ...

Reply:
Fixed, thanks!